extensions/wikia/WikiFactory/tests/WikiFactoryTest.php

```diff
@@ -223,6 +241,13 @@ public function testPrepareUrlToParseDataProvider() {
 		];
 	}
 
+	public function testGetHostById() {
+		$this->mockStaticMethod( 'WikiFactory', 'getVarValueByName', 1 );
+		$this->mockStaticMethod( 'WikiFactory', 'getLocalEnvURL', 'test_host/' );
+
+		$this->assertEquals( 'test_host', WikiFactory::getHostById( 2 ) );
```

Comment:
getHostById is also used in other places. I'd like to avoid refactoring it at this moment.